fuelup-init.sh

```diff
@@ -76,6 +78,36 @@ main() {
         esac
     done
 
+    if [ "$prompt_modify" = "yes" ]; then
+        case $SHELL in
+            */bash)
+                SHELL_PROFILE=$HOME/.bashrc
+                ;;
+            */zsh)
+                SHELL_PROFILE=$HOME/.zshrc
+                ;;
+            */fish)
+                SHELL_PROFILE=$HOME/.config/fish/config.fish
+                ;;
+            *)
+                err "Failed to detect shell; please add ${FUELUP_DIR}/bin to your PATH manually."
```

Comment:
This will always fail here for Windows user with no indication of how to proceed. It should instead either tell the user to use `--no-modify-path`, or continue to not attempt to modify PATH.

Reply:
I updated it to be a warning instead. Also, wouldn't a Windows user already fail at the `uname` step?